tests/fixtures/optimizer/qualify_tables.sql

```diff
@@ -168,7 +168,7 @@ SELECT x FROM c.db.t AS t, LATERAL UNNEST(t.xs) AS "_0";
 
 # title: table with ordinality
 SELECT * FROM t CROSS JOIN JSON_ARRAY_ELEMENTS(t.response) WITH ORDINALITY AS kv_json;
-SELECT * FROM c.db.t AS t CROSS JOIN JSON_ARRAY_ELEMENTS(t.response) WITH ORDINALITY AS kv_json;
+SELECT * FROM c.db.t AS t CROSS JOIN JSON_ARRAY_ELEMENTS(t.response) WITH ORDINALITY AS "_0"(kv_json);
```

Comment:
So yeah, I would avoid doing this. I'm afraid it won't be correct in some cases, despite possibly being correct for `JSON_ARRAY_ELEMENTS` in particular.